Build failed. X11 not linked?

I was building `rocm-opencl-runtime` when it failed. The build command is `make local`, executed in `/rocm-opencl-runtime`.

```
[9/132] Building CXX object api/opencl/amdocl/CMakeFiles/amdocl64.dir/cl_gl.cpp.o
FAILED: api/opencl/amdocl/CMakeFiles/amdocl64.dir/cl_gl.cpp.o 
/usr/lib64/ccache/bin/x86_64-solus-linux-g++  -DATI_ARCH_X86 -DATI_BITS_64 -DATI_COMP_GCC -DATI_OS_LINUX -DCL_USE_DEPRECATED_OPENCL_1_0_APIS -DCL_USE_DEPRECATED_OPENCL_1_1_APIS -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DCL_USE_DEPRECATED_OPENCL_2_0_APIS -DCOMGR_DYN_DLL -DLINUX -DLITTLEENDIAN_CPU -DOPENCL_EXPORTS -DOPENCL_MAJOR=2 -DOPENCL_MINOR=0 -DUNIX_OS -DUSE_COMGR_LIBRARY -DVEGA10_ONLY=false -DWITH_AQL -DWITH_HSA_DEVICE -DWITH_LIGHTNING_COMPILER -DWITH_ONLINE_COMPILER -DWITH_TARGET_AMDGCN -D__AMD64__ -D__x86_64__ -Damdocl64_EXPORTS -DqLittleEndian -I/usr/lib64/llvm/roc/include -I/usr/lib64/llvm/roc/include/clang -I/usr/lib64/llvm/roc/include/lld -I/usr/lib64/llvm/roc/include/llvm/Target/AMDGPU -I/usr/include/hsa -I../runtime -I../runtime/include -I../api/opencl -I../api/opencl/khronos -I../api/opencl/khronos/headers -I../api/opencl/khronos/headers/opencl2.2 -I../compiler/lib -I../compiler/lib/include -I../compiler/lib/backends/common -I../compiler/lib/loaders -I../compiler/lib/loaders/elf/utils/libelf -I../compiler/lib/loaders/elf/utils/common -I../compiler/tools -mtune=generic -march=x86-64 -g2 -O2 -pipe -fPIC -D_FORTIFY_SOURCE=2 -fstack-protector-strong --param=ssp-buffer-size=32 -fasynchronous-unwind-tables -ftree-vectorize -feliminate-unused-debug-types -Wall -Wno-error -Wp,-D_REENTRANT -std=c++11 -O2 -g -DNDEBUG -fPIC -MD -MT api/opencl/amdocl/CMakeFiles/amdocl64.dir/cl_gl.cpp.o -MF api/opencl/amdocl/CMakeFiles/amdocl64.dir/cl_gl.cpp.o.d -o api/opencl/amdocl/CMakeFiles/amdocl64.dir/cl_gl.cpp.o -c ../api/opencl/amdocl/cl_gl.cpp
In file included from ../api/opencl/khronos/headers/EGL/egl.h:36,
                 from ../api/opencl/amdocl/cl_gl.cpp:20:
../api/opencl/khronos/headers/EGL/eglplatform.h:99:10: fatal error: X11/Xlib.h: No such file or directory
   99 | #include <X11/Xlib.h>
      |          ^~~~~~~~~~~~
compilation terminated.
```

X11 is installed on the system (specifically `libx11` and `libx11-devel`). All other dependencies according to `builddeps.jpg` have been successfully built, installed, and copied to the local repository.

It seems to me that the compilation command isn't linking X11 (I don't see `-lX11` in there).

The full output of the command can be found here: https://gist.github.com/KonaeAkira/a891369d52496d2ed22f135418832394

--- a/rocm-opencl-runtime/package.yml
+++ b/rocm-opencl-runtime/package.yml
@@ -16,6 +16,7 @@ patterns   :
     - /*
 extract    : False
 builddeps  :
+    - libx11-devel
     - pkgconfig(OpenCL)
     - pkgconfig(gl)
     - pkgconfig(libhsakmt)

